server/models.js

```diff
@@ -12,8 +12,8 @@ connection.connect();
 
 module.exports = {
   users: (fakeUser) => {
-    const queryVal = [fakeUser.name];
-    const quuery = 'INSERT INTO USERS (name) VALUES(?)';
+    const queryVal = [fakeUser.name, fakeUser.image];
+    const quuery = 'INSERT INTO USERS (name, image) VALUES(?, ?)';
```

Comment:
It might help to maybe call it queryString or something more distinct instead of adding an extra 'u'. That can be confusing to read.

Reply:
Since these are the query values, I thought it make more sense to call them queryVal. Thank you for comment though.